.gitignore

```diff
@@ -0,0 +1,15 @@
+.classpath
+.jekyll-cache/
+.project
+_site/
+Gemfile.lock
+
+## VS Code
+
+.settings/
+.vscode/
+
+## Inproper copy when switching branch
+
+target/
+src/
```

Comment:
Hi, can this _.gitignore_ file be moved in another PR?.

Reply:
https://github.com/apache/systemml/pull/879

Reply:
Also removed it from this pull request.